### ISSUE TITLE
Gusenberg Template

### ISSUE THREAD

**Issue by Duderocktv** [rubbertoe98/vWeaponsToolkit#40]
So many thompsons available. Any chance of making a Gusenberg template

**Comment by Hxrv3y** [comment]
> So many thompsons available. Any chance of making a Gusenberg template

Yes I have done that on my fork of the software. Check my repos I frequently update mine still
